src/components/SubscribeButton/SubscribeButton.tsx

```diff
@@ -34,11 +33,7 @@ const _SubscribeButton = ({
           onClick={onSubscribeButtonClicked}
           loading={loading}
           icon={<SubscribeButtonIcon isSubscribed={isSubscribed} />}
-        >
-          <span className="font-base text-sm">
-            <Trans>Get Updates</Trans>
```

Comment:
Removed this text to reduce clutter in the header after adding the second Bookmark button